pytest_doctestplus/plugin.py

```diff
@@ -11,18 +11,41 @@
 import re
 import sys
 import warnings
+from textwrap import indent
 
```

Comment:
as @pllim said, we should either drop support for python 2 or suffer. In py2 indent can be added using text wrapper class: https://docs.python.org/2.7/library/textwrap.html#textwrap.TextWrapper

Reply:
I added a quick workaround for now but agree that we'll want to drop Python 2 shortly.